### ISSUE TITLE
CVE-2020-8203 is not modified in /.internal/baseSet.js

### ISSUE THREAD

**Issue by reeser** [lodash/lodash#4874]
https://github.com/lodash/lodash/blob/86a852fe763935bb64c12589df5391fd7d3bb14d/.internal/baseSet.js#L29

## issue:
https://nvd.nist.gov/vuln/detail/CVE-2020-8203

## description:
the issue is modified in lodash.js since 4.17.17, but  not modified when use lodash/zipObjectDeep util now(4.17.19).  Could anyone modify it? Thanks~

```js
const _ = require('lodash');
const zipObjectDeep = require('lodash/zipObjectDeep');

// this is ok from 4.17.17
//const result = _.zipObjectDeep(['a.b.__proto__.c'], [123]);

// this is not ok util now 4.17.19
const result = zipObjectDeep(['a.b.__proto__.c'], [123]);
console.log(c);

const test = {
  "a": 1
}
for (const attr in test) {
  console.log('attr in test', attr);
}
```

the result is:
```shell
123
attr in test a
attr in test c
```

**Comment by bnjmnt4n** [comment]
This has been addressed in v4.17.20.

**Comment by orionleeMS** [comment]
@bnjmnt4n could you link to the commit where this was address? I was looking at the diffs between v4.17.19 and v4.17.20 and I am having trouble finding where it was addressed.

https://cdn.jsdelivr.net/npm/lodash@4.17.19/lodash.js
https://cdn.jsdelivr.net/npm/lodash@4.17.20/lodash.js
